templates/web/base/reports/_list-filters.html

```diff
@@ -9,9 +9,21 @@
         <option value="shortlisted"[% ' selected' IF filter_status.shortlisted %]>[% loc('Shortlisted') %]</option>
         <option value="unshortlisted"[% ' selected' IF filter_status.unshortlisted %]>[% loc('Unshortlisted') %]</option>
       [% END %]
+      [% IF c.user_exists AND c.user.is_superuser OR c.user.belongs_to_body(body.id) %]
+        <option value="open"[% ' selected' IF filter_status.open %]>[% loc('New reports') %]</option>
+        <option value="investigating"[% ' selected' IF filter_status.investigating %]>[% loc('Investigating') %]</option>
+        <option value="in progress"[% ' selected' IF filter_status.in_progress %]>[% loc('In progress') %]</option>
+        <option value="action scheduled"[% ' selected' IF filter_status.action_scheduled %]>[% loc('Scheduled reports') %]</option>
```

Comment:
Just to keep translation easy (and I don't think it really changes how this is used), could these be the states alone, e.g. "Action Scheduled" or "No further action", without 'reports' on the end. I don't think a staff user will mind it like that.
(Also, thinking ahead, if these states are going into the database.)